substrate/frame/revive/src/exec.rs

```diff
@@ -1286,6 +1298,13 @@ where
 	fn account_balance(&self, who: &T::AccountId) -> U256 {
 		T::Currency::reducible_balance(who, Preservation::Preserve, Fortitude::Polite).into()
 	}
+
+	/// Certain APIs, e.g. `{set,get}_immutable_data` behave differently depending
+	/// on the configured entry point. Thus, we allow setting the export manually.
+	#[cfg(all(feature = "runtime-benchmarks", feature = "riscv"))]
```

Comment:
why not just feature = "runtime-benchmarks"?

Reply:
Yeah, there are also many other feature gates that require both.